Day_2_Software_engineering_best_practices/README.md

```diff
@@ -124,12 +124,18 @@ Some theory:
 
 **Exercise:**
 
-* make package with setup.py
+We created `setup.py` and `spectral_analysis/version.py` to make the a
```

Comment:
I don't know. I just saw that you do it in pandas but I did not see it in scikit-learn or the template. Is there any reason to go for it?